presto-main/src/main/java/com/facebook/presto/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -201,6 +203,7 @@
     private final SqlFunctionProperties sqlFunctionProperties;
     private final List<Expression> parameters;
     private final WarningCollector warningCollector;
+    private final Optional<Integer> maxInListValues;
```

Comment:
The more consistent naming is probably `maxInListValueCount` or `maxInListLength`?